thentos-core/purescript/.gitignore

```diff
@@ -0,0 +1,5 @@
+/bower_components/
+/node_modules/
+/output/
+/.psci*
+/src/.webpack.js
```

Comment:
does this still exist?  that would probably mean that somebody around here is already using webpack...